packages/bridge-controller/src/utils/balance.test.ts

```diff
@@ -10,7 +10,6 @@ import { fetchTokenBalance } from './balance';
 import { FakeProvider } from '../../../../tests/fake-provider';
 
 declare global {
-  // eslint-disable-next-line no-var
```

Comment:
This is a mystery. I wasn't able to figure out why this got flagged as an unused directive.